Tests/Twig/Extension/SonataAdminExtensionTest.php

```diff
@@ -1218,20 +1218,22 @@ class="x-editable"
 
             array(
                 '<td class="sonata-ba-list-field sonata-ba-list-field-text" objectId="12345">
-                <div class="sonata-readmore" data-readmore-height="40" data-readmore-more="Read more" data-readmore-less="Close">A very very very very very very very very very long string.</div>
+                <div class="sonata-readmore" data-readmore-height="40" data-readmore-more="Read more" '.
+                'data-readmore-less="Close">A very very long string.</div>
```

Comment:
There is [an example](https://github.com/nykopol/SonataAdminBundle/blob/526b2e731bfa817f8db83da3e9b2b7c9739990f8/Tests/Twig/Extension/SonataAdminExtensionTest.php#L656) in the file already